ocaml-lsp-server/src/inference.ml

```diff
@@ -39,23 +39,28 @@ let language_id_of_fname s =
 
 let force_open_document (state : State.t) uri =
   let filename = Uri.to_path uri in
-  let text = Io.String_path.read_file filename in
-  let languageId = language_id_of_fname filename in
-  let text_document =
-    TextDocumentItem.create ~uri ~languageId ~version:0 ~text
-  in
-  let params = DidOpenTextDocumentParams.create ~textDocument:text_document in
-  let+ doc =
-    Document.make (State.wheel state) state.merlin_config
-      ~merlin_thread:state.merlin params
-  in
-  Document_store.put state.store doc;
-  doc
+  match Io.String_path.read_file filename with
+  | exception Sys_error _ ->
+    Log.log ~section:"warning" (fun () ->
+        Log.msg "Unable to open file" [ ("filename", `String filename) ]);
```

Comment:
is the section expected to be "warning"?

Reply:
I suppose it is totally expected. I made it debug